### PR TITLE
Integrate ConfimationModal into code

### LINKED ISSUE #322: Incorporate ConfirmationModal into Components
Instead of copy and pasting a dumb modal every time we want to confirm something let's do this.

[`src/Pages/Overview/OverviewProfile.js`](https://github.com/SCE-Development/Core-v4/blob/fb8cd2d47613aff84082b79c0ab8ff383d3a5929/src/Pages/Overview/OverviewProfile.js#L72)
```jsx
{/*  at the top import */}
import ConfirmationModal from '../path_here'

{/*  Line 57 */}
<ConfirmationModal
  headerText={`Delete ${props.user.name} ?`}
  bodyText={`Are you sure you want to delete ${props.user.name}? They're kinda cute and
          they'll be gone forever if you do`}
  confirmText={`Yes, ${props.user.name} is dead to me`}
  cancelText={"No, they're chill"}
  toggle={() => setToggleDelete(!toggleDelete)}
  handleConfirmation={() => {
              props.deleteUser(props.user)
              setToggleDelete(!toggleDelete)
            }}
  open={toggleDelete}
/>
```

### PR DESCRIPTION
**Pull request by kalyanpullela**
Resolves #322

### CODE REVIEW

File: src/Pages/Overview/OverviewProfile.js [hunk 1]
```diff
@@ -69,35 +70,19 @@ export default function OverviewProfile (props) {
         <InfoCard user={props.user} token={props.token} />
       </Modal>
 
-      <Modal
-        isOpen={toggleDelete}
+      <ConfirmationModal
+        headerText={`Delete ${props.user.name} ?`}
```
Comment (evanugarte) [inline src/Pages/Overview/OverviewProfile.js]: the modal looks like this when we run it
image

File: src/Pages/EventManager/EventManagerModal.js [hunk 2]
```diff
@@ -146,15 +147,13 @@ function EventManagerModal (props) {
...
-        <ModalFooter>
-          <Button onClick={toggleConfirmationModal}>Cancel</Button>
-          <Button color='danger' onClick={handleDeletion}>
-            Yes, delete
-          </Button>
-        </ModalFooter>
-      </Modal>
+      <ConfirmationModal
+        headerText={`Delete ${props.title} ?`}
+        bodyText='The event will be gone forever if you do this'
```
Comment (evanugarte) [inline src/Pages/EventManager/EventManagerModal.js]:
```suggestion
        bodyText='The event will be gone forever if you do this.'
```
image

File: src/Pages/2DPrinting/2DPrinting.js [hunk 3]
```diff
@@ -327,39 +328,20 @@ export default function Printing (props) {
             </Button>
           </ModalFooter>
 
-          <Modal
-            isOpen={confirmModal}
+          <ConfirmationModal
+            headerText='Are you sure you want to print?'
```
Comment (ThaiQ) [inline src/Pages/2DPrinting/2DPrinting.js]: @evanugarte @kalyanpullela 
Are these good with the way they are?
or should we make another object/dictionary for these fields?
Ex:
```js
let confirmModalMsg= {
headerText: 'Are you sure you want to print?',
bodyText: 'Click Yes or Go Back',
...
}
```

and call it in the modal:
```js
<ConfirmationModal
headerText= confirmModalMsg.headerText
bodyText= confirmModalMsg.bodyText
.../>
```

**Comment by ThaiQ** [comment]
@kalyanpullela could you provide screenshots the changes?

File: src/Pages/2DPrinting/2DPrinting.js (hunk 3 above)
Comment (ThaiQ) [reply src/Pages/2DPrinting/2DPrinting.js]: same goes for the other 2 files

Comment (evanugarte) [reply src/Pages/2DPrinting/2DPrinting.js]: Ahhh good idea. You can also do

```jsx
const confrimModalProps = { /* stufff */ }


<ConfirmationModal
  {...confirmModalProps}
```

**Comment by evanugarte** [comment]
@ThaiQ 
image

image

**Pull request merged**